test/test_quantization.py

```diff
@@ -793,6 +813,31 @@ def checkQuantized(model):
         model = quantize_qat(model, test_only_train_fn, self.img_data)
         checkQuantized(model)
 
+    def test_train_save_load_eval(self):
```

Comment:
Should this function be called test_train_save_load_qat(self):?

Reply:
The class has information about PTQ vs QAT  - `EagerModeQuantizationAwareTrainingTest`